src/conditional-flow/multiple-conditions.js

```diff
@@ -1,19 +1,15 @@
 // This function should return true if num is greater
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
-function isInRange (num, lower, upper) {
-
-  // TODO: write code in this function body to pass the tests
-
+function isInRange(num, lower, upper) {
+  return num >= lower && num <= upper ? true : false;
 }
 
 // This function should return true if the passed string is equal
 // to "Hello" or "Goodbye". Implement this with a single
 // if statement.
-function isHelloOrGoodbye (val1) {
-
-  // TODO: write code in this function body to pass the tests
-
+function isHelloOrGoodbye(val1) {
+  return (val1 === "Hello") | (val1 === "Goodbye") ? true : false;
```

Comment:
Just a small note on using bitwise operators - they're not always the most readable things to use. You could have used a logical OR instead like this:
```js
return (val1 === 'Hello') || (val1 === 'Goodbye')
```

Which I think is more readable